docs/reference/interfaces/ScriptContext.md

```diff
@@ -2,9 +2,8 @@
 
 # Interface: ScriptContext
 
-ScriptContext contains all the information in the transaction's [preimage][https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/abc/replay-protected-sighash.md#digest-algorithm](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/abc/replay-protected-sighash.md#digest-algorithm).
-The preimage is automatically generated during the user's construction of the transaction,
-and the user does not need to calculate it explicitly
+ScriptContext contains all the information in the transaction's [preimage](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/abc/replay-protected-sighash.md#digest-algorithm).
```

Comment:
there are many broken links here.

i can fix them, but wanted to check with @xhliu first, in case these pages were generated from code somehow (meaning, it wouldn't be worth fixing them by hand)

Reply:
Yeah, auto generated.